crates/wildbg-c/src/lib.rs

```diff
@@ -118,17 +117,18 @@ impl From<BgMove> for CMove {
         let mut c_move = CMove::default();
         #[allow(clippy::len_zero)]
         if details.len() > 0 {
-            c_move.detail1 = (&details[0]).into();
+            c_move.moves[0] = (&details[0]).into();
```

Comment:
Now that we have an array, we don't need all those if statements.
We could use
```Rust
c_move_moves.iter_mut().zip(details).for_each
```
or
```Rust
details.iter().enumerate().for_each(
```

Personally I would use the `enumerate` variant to avoid a closure with two very similar arguments (`detail`), but that's a matter of taste.

Reply:
nice! I've changed this to use enumerate.